tests/scripts/minishift.sh

```diff
@@ -36,7 +36,6 @@ MEMORY=${MEMORY:-"3000"}
 
 case "${1:-}" in
   up)
-    # Use kubeadm bootstrapper for 1.9+ since localkube was deprecated in 1.8
     echo "starting minishift with localkube bootstrapper"
```

Comment:
shouldn't this comment get updated to say kubeadm bootstrapper, not localkube bootstrapper?

Reply:
true. i'll just simplify it to "starting minishift" since the method choice is old news